### PR TITLE
Added my current used filaments from my spoolman instance

### PR DESCRIPTION
**Pull request by Disane87**
Added my current used filaments from my spoolman instance

### CODE REVIEW

**Comment by Disane87** [comment]
@Donkie I have created a Script whoch gathers the flaments from the API. Do you mind when I add this to the repo?

**Comment by Donkie** [comment]
That's an interesting idea, though I think it's better if people in general grab information from the manufacturer website, since then the info will be more complete. Your script only populates a single weight item for example, while the supplier may well sell multiple sizes.

You may use your script but please ensure the info is complete for each filament you add.

**Comment by Disane87** [comment]
> That's an interesting idea, though I think it's better if people in general grab information from the manufacturer website, since then the info will be more complete. Your script only populates a single weight item for example, while the supplier may well sell multiple sizes.
Yeah thats one where I'm currently working on. But actually I don't have any other siztes so I haven't done it yet. 

> You may use your script but please ensure the info is complete for each filament you add.
Yeah sure! Since I add the filaments correctly to spoolman by infos of the manufacturer there should be no problem from my side.

The script is also inteded to create a base from your spoolman and than add more details (manually) if needed.
It's just an POC to get more traction into the process of adding filaments.

**Review by Donkie — CHANGES_REQUESTED** [review]

File: filaments/Jayo.json [hunk 1]
```diff
@@ -0,0 +1,139 @@
+{
+    "manufacturer": "Jayo",
+    "filaments": [
+        {
+            "name": "Jayo {color_name}",
```
[the remark this answers is not in the document]
Comment (Disane87) [reply filaments/Jayo.json]: Did that and renamed files to lowercase

File: filaments/prusament.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+    "manufacturer": "prusament",
```
Comment (Donkie) [inline filaments/prusament.json:2]: Uppercase P

File: filaments/jayo.json [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+                    "hex": "B272C9"
+                },
+                {
+                    "name": "Matte Black",
+                    "hex": "000000"
+                }
+            ]
+        },
+        {
+            "name": "Jayo {color_name}",
```
Comment (Donkie) [inline filaments/jayo.json:105]: Still have company name on some of these

File: filaments/jayo.json [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+                {
+                    "name": "Orange",
+                    "hex": "FC9036"
+                },
+                {
+                    "name": "Blue",
+                    "hex": "1241CD"
+                },
+                {
+                    "name": "Gras-Green",
```
Comment (Donkie) [inline filaments/jayo.json:49]: Grass Green I presume?

File: filaments/jayo.json [hunk 5]
```diff
@@ -0,0 +1,139 @@
+{
+    "manufacturer": "Jayo",
```
Comment (Donkie) [inline filaments/jayo.json:2]: Should be JAYO right?

File: filaments/creality.json [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+            ],
+            "extruder_temp": 210,
+            "bed_temp": 60,
+            "colors": [
+                {
+                    "name": "Hyper - White",
+                    "hex": "FFFFFF"
+                },
+                {
+                    "name": "Hyper - Grey",
```
Comment (Donkie) [inline filaments/creality.json:25]: Hyper should be added to the filament name instead

**Review by Donkie — CHANGES_REQUESTED** [review]

**Comment by Disane87** [comment]
Sorry I shouldn't have done these changes with that little review from my side. Totally was my fault! Changed the things you've requested

**Pull request merged**